internal/event/target/webhook.go

```diff
@@ -196,7 +196,7 @@ func (target *WebhookTarget) send(eventData event.Event) error {
 	if err != nil {
 		return err
 	}
-	defer xioutil.DiscardReader(resp.Body)
+	defer xhttp.DrainBody(resp.Body)
```

Comment:
Thanks for catching this @jiuker, however it looks like this was changed only two weeks ago for a NUMA thing; maybe we can manually close this body here

/cc @harshavardhana

Reply:
No this was a mistake @vadmeste xhttp.DrainBody is correct to use here.